wishlist.css

```diff
@@ -0,0 +1,156 @@
+:root{
+    --border-color:#d1d1d1;
+    --table-element-color:#fb923c;
+    --primary-color: #94a3b8;
+    --white:rgb(255, 255, 255);
+    }
+
+/*wishlist-page*/
+
+
+
+    *{
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+        
+    }
```

Comment:
This is used repeatedly so you should add  this in utility css file (if you've made it).